.vscode/launch.json

```diff
@@ -1,11 +1,11 @@
 {
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "command": "bun dev",
-            "name": "Run Riven Frontend",
-            "request": "launch",
-            "type": "node-terminal"
-        }
-    ]
-}
\ No newline at end of file
+	"version": "0.2.0",
```

Comment:
oh this is useful! awesome! This does use `bun` though, thought Mini mentioned it was unstable?

Reply:
I use bun for everything. But it's probably more useful if we replace it with npm (or pnpm) as more people have that set up. What do you think?

Reply:
I'd say pnpm, but the performance that `bun` boasts about looks really nice..

Reply:
> I'd say pnpm, but the performance that `bun` boasts about looks really nice..

Bun is faster but one weird package and it might break, i can look into it

Reply:
Svelte 5 lookin goood too! 🔥